Infrastructure/Kubernetes/05-kubernetes-networking.md

```diff
@@ -0,0 +1,18 @@
+# Introduction
+
+[Kubernetes Networking Model](https://kubernetes.io/docs/concepts/cluster-administration/networking/#how-to-achieve-this)
+[Services](https://kubernetes.io/docs/concepts/services-networking/service/)
+[DNS Pods](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/)
+[Connecting Apps and Services](https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/)
+
+## Test
+
+What are the 4 types of services in kubernetes?
+
+If you have a service named 'example-svc' in the default namespace, what will be its fully qualified domain name be within the cluster?
+
+What command should I run to get the ip address of the hello-minikube pod running on the minikube cluster?
```

Comment:
It may be better to ask what commands _can_ I run since there may be multiple ways to figure something like this out.